doc/COMPILE.md

```diff
@@ -606,6 +606,27 @@ running cmake like to enable debug symbol.
 For Gentoo it's recommended to emerge sys-apps/lsb-release to name the package
 right even thou not required.
 
+### With Flatpak (Beta)
+
+It is also possible to build a Flatpak package containing Elektra:
+
+```bash
+# install flatpak and flatpak-builder, using the package manager of your distro
+$ apt install flatpak flatpak-builder
+
```

Comment:
`flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo` was missing for me here.

Reply:
Thanks for the comment! I have added it to the docs.